nix/keys.nix

```diff
@@ -24,11 +24,11 @@ let
       default = null;
       type = types.nullOr types.path;
       description = ''
-        When non-null, contents of the specified file will be deployed to
-        the specified key on the target machine.  If the key name is
+        When non-null, contents of the specified file will be deployed to the
+        specified key on the target machine.  If the key name is
         <replaceable>password</replaceable> and <literal>/foo/bar</literal> is set
         here, the contents of the file
-        <filename>/run/keys/<replaceable>password</replaceable></filename>
+        <filename><replaceable>destinationFolder</replaceable>/<replaceable>password</replaceable></filename>
```

Comment:
I'd love `destDir` more :)  The shorter, the better. No "folders" at least. Child trauma.